util/util.go

```diff
@@ -191,41 +200,62 @@ func (s server) GetGRPCServer(opts ...grpc.ServerOption) *grpc.Server {
 	return grpc.NewServer(opts...)
 }
 
-func readContent(path string) error {
-	cont, err := os.Open(path)
+func InitilizegRPCServer(conf *model.Config) (*server, error) {
+
+	store, err := database.NewStore(conf)
+
 	if err != nil {
-		log.Fatal(err)
-		return err
+		return nil, err
 	}
-	_, err = io.Copy(os.Stdout, cont)
-	if err != nil {
-		log.Fatal(err)
-		return err
+
+	s := &server{
+		store:   store,
+		auth:    NewAuthenticator(conf.SigninKey, conf.AuthKey),
+		tls:     conf.TLS.Enabled,
 	}
-	return nil
+	return s, nil
 }
 
+func NewConfigFromFile(path string) (*model.Config, error) {
+	f, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+
+	var c model.Config
+	err = yaml.Unmarshal(f, &c)
+	if err != nil {
+		return nil, err
+	}
 
+	if c.Host == "" {
+		log.Println("Host not provided in the configuration file")
+		c.Host = "localhost:50051"
+	}
 
-func InitilizegRPCServer() *server {
+	if c.SigninKey == "" {
+		log.Println("SigninKey not provided in the configuration file")
+		c.Host = "dev-env"
+	}
 
-	store, err := database.NewStore()
+	if c.AuthKey == "" {
+		log.Println("AuthKey not provided in the configuration file")
+		c.Host = "development-environment"
+	}
 
-	if err != nil {
-		log.Fatalf("failed to connect to database: %v", err)
+	if c.DB.Host == "" || c.DB.User == "" || c.DB.Pass == "" || c.DB.Name == "" {
+		return nil, errors.New("DB paramenters missing in the configuration file")
 	}
```

Comment:
I think, a default value could be setup by informing user.

Reply:
why 👎🏽

Reply:
Because the user should have a good configuration file. Its easier to throw an error, the user in case need just to modify the config file.

Reply:
Ok, I agree, but I do not think they need to change config file, you can the values in config as default ones then win-win . Am I wrong ?

Reply:
Defaut value will throw an error anyway cause you can not connect to DB.